pkgs/development/libraries/portaudio/default.nix

```diff
@@ -8,7 +8,8 @@
 , AudioToolbox
 , CoreAudio
 , CoreServices
-, Carbon }:
+, Carbon
+, withALSA ? stdenv.hostPlatform.isLinux && !stdenv.isDarwin }:
```

Comment:
So cross building works with a darwin host?

Reply:
I don't know. I only tested cross-compiling on Linux for Windows, and it did work.

It's been a while since I've touched this, but it seems like it should be checking the target platform is not Windows, rather than checking that the host platform is Linux. Is that possible?

Reply:
I think there should be `isWindows` predicates for any kind of platform.